contracts/extensions/VotingReputation/VotingReputation.sol

```diff
@@ -18,28 +18,17 @@
 pragma solidity 0.7.3;
 pragma experimental ABIEncoderV2;
 
-import "./../colonyNetwork/IColonyNetwork.sol";
-import "./../colony/ColonyRoles.sol";
-import "./../common/BasicMetaTransaction.sol";
-import "./../common/ERC20Extended.sol";
-import "./../patriciaTree/PatriciaTreeProofs.sol";
-import "./../tokenLocking/ITokenLocking.sol";
-import "./ColonyExtension.sol";
-
-
-contract VotingReputation is ColonyExtension, PatriciaTreeProofs, BasicMetaTransaction {
-
-  // Events
-  event MotionCreated(uint256 indexed motionId, address creator, uint256 indexed domainId);
-  event MotionStaked(uint256 indexed motionId, address indexed staker, uint256 indexed vote, uint256 amount);
-  event MotionVoteSubmitted(uint256 indexed motionId, address indexed voter);
-  event MotionVoteRevealed(uint256 indexed motionId, address indexed voter, uint256 indexed vote);
-  event MotionFinalized(uint256 indexed motionId, bytes action, bool executed);
-  event MotionEscalated(uint256 indexed motionId, address escalator, uint256 indexed domainId, uint256 indexed newDomainId);
-  event MotionRewardClaimed(uint256 indexed motionId, address indexed staker, uint256 indexed vote, uint256 amount);
-  event MotionEventSet(uint256 indexed motionId, uint256 eventIndex);
-
-  // Constants
+import "./../../colonyNetwork/IColonyNetwork.sol";
+import "./../../colony/ColonyRoles.sol";
+import "./../../common/BasicMetaTransaction.sol";
+import "./../../common/ERC20Extended.sol";
+import "./../../patriciaTree/PatriciaTreeProofs.sol";
+import "./../../tokenLocking/ITokenLocking.sol";
+import "./../ColonyExtension.sol";
+import "./VotingReputationDataTypes.sol";
+
+
+contract VotingReputation is ColonyExtension, PatriciaTreeProofs, BasicMetaTransaction, VotingReputationDataTypes {
   uint256 constant UINT128_MAX = 2**128 - 1;
```

Comment:
I would put the constants in `DataTypes` also

Reply:
![out](https://user-images.githubusercontent.com/311812/186885725-0c5fe547-13e0-4637-97d7-7aafa8533c15.png)